tests/subresource_api_test.go

```diff
@@ -41,6 +41,8 @@ var _ = Describe("Subresource Api", func() {
 	virtCli, err := kubecli.GetKubevirtClient()
 	tests.PanicOnError(err)
 
+	manual := v1.RunStrategyManual
```

Comment:
we need functional tests for restartOnError

Reply:
There's some overlap between tests here by the way. There are already redundant tests in vm_test.go under the "Using RunStrategyRestartOnError" context.

But it can never hurt to have more tests. :)